release/release_tests.yaml

```diff
@@ -619,6 +619,25 @@
     type: sdk_command
     file_manager: sdk
 
+- name: ray-data-bulk-ingest-file-size-oom-test
+  group: AIR tests
+  working_dir: air_tests/air_benchmarks/mlperf-train
+
+  stable: true
+  env: staging
+
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: app_config_oom.yaml
+    cluster_compute: compute_cpu_16.yaml
+
+  run:
+    timeout: 1800
+    script: NUM_IMAGES_PER_FILE="6144" bash file_size_benchmark.sh
```

Comment:
What are your thoughts on merging the two release tests vs. keeping them separate? Once the OOM killer is ready, why not have just one test?

Reply:
should we keep one that doesn't oom for regression ? They could track different regression? Since its single node it shoudn't be that expensive

Reply:
Let's have a single test that covers a small file size, a medium (non-OOM) file size, and then a large filesize which will OOM. We can remove `2048` (the largest we could run previously) and replace it with your larger file size.

Reply:
(We already have a lot of tests 😅 )